gvm/protocols/gmpv9/__init__.py

```diff
@@ -0,0 +1,443 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2018 - 2019 Greenbone Networks GmbH
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+# pylint: disable=arguments-differ, redefined-builtin, too-many-lines
+
+"""
+Module for communication with gvmd in `Greenbone Management Protocol version 9`_
+
+.. _Greenbone Management Protocol version 9:
+    https://docs.greenbone.net/API/GMP/gmp-9.0.html
+"""
+import collections
+import numbers
+
+from enum import Enum
+from typing import Any, List, Optional
+
+from gvm.errors import InvalidArgument, RequiredArgument
+from gvm.utils import deprecation
+from gvm.xml import XmlCommand
+
+from gvm.protocols.gmpv8 import Gmp as Gmpv8, _to_bool, _add_filter
+from gvm.protocols.gmpv7 import _is_list_like, _to_comma_list
+
+from . import types
+from .types import *
+from .types import _UsageType as UsageType
+
+PROTOCOL_VERSION = (9,)
+
+
+class Gmp(Gmpv8):
+
+    types = types
+
+    @staticmethod
+    def get_protocol_version() -> tuple:
+        """Determine the Greenbone Management Protocol version.
+
+        Returns:
+            tuple: Implemented version of the Greenbone Management Protocol
+        """
+        return PROTOCOL_VERSION
+
+    def create_audit(
+        self,
+        name: str,
+        config_id: str,
+        target_id: str,
+        scanner_id: str,
+        *,
+        alterable: Optional[bool] = None,
+        hosts_ordering: Optional[HostsOrdering] = None,
+        schedule_id: Optional[str] = None,
+        alert_ids: Optional[List[str]] = None,
+        comment: Optional[str] = None,
+        schedule_periods: Optional[int] = None,
+        observers: Optional[List[str]] = None,
+        preferences: Optional[dict] = None
+    ) -> Any:
+        """Create a new audit task
+
+        Arguments:
+            name: Name of the task
+            config_id: UUID of scan config to use by the task
```

Comment:
Internally it is still a scan config but we call it policy now
```suggestion
            config_id: UUID of policy to use by the task
```